### ISSUE TITLE
Form error messages use unavailable glyphicons

### ISSUE THREAD

**Issue by Guite** [zikula/core#3243]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.4.4-git
| PHP Version       | 7.x

### Expected behavior
Form error messages are properly displayed.

### Actual behavior
Messages are shown like this:
```
<li>
<span class="glyphicon glyphicon-exclamation-sign"></span>
This value should be of type integer.
</li>
```
whereby the icon is not shown correctly, because our custom bootstrap file does not include glyph icons.

### Steps to reproduce
Submit a form with invalid data.

### Proposal
Override form templates in our `FormExtensionBundle` replacing glyphicons by FontAwesome.

Original template in `vendor/symfony/symfony/src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig`:
```
{% block form_errors -%}
    {% if errors|length > 0 -%}
    {% if form.parent %}<span class="help-block">{% else %}<div class="alert alert-danger">{% endif %}
    <ul class="list-unstyled">
        {%- for error in errors -%}
            <li><span class="glyphicon glyphicon-exclamation-sign"></span> {{ error.message }}</li>
        {%- endfor -%}
    </ul>
    {% if form.parent %}</span>{% else %}</div>{% endif %}
    {%- endif %}
{%- endblock form_errors %}
```

**Comment by craigh** [comment]
I think this should also be reported at symfony. They shouldn't have this dependency in place at all.

your proposal is ok, except maybe the icon should just be completely removed and not replaced?

**Comment by Guite** [comment]
Reported in https://github.com/symfony/symfony/issues/20742

**Comment by Guite** [comment]
Fixed in #3244
